feign/src/main/resources/logback_spring.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
```

Comment:
do we need this file for our articles?

Reply:
That file was before, I just tried to align with the latest version and I updated the file because failed at start up.